woocommerce-services.php

```diff
@@ -651,12 +652,17 @@ public function woocommerce_payment_gateways( $payment_gateways ) {
 		 * Registers the React UI bundle
 		 */
 		public function admin_enqueue_scripts() {
-			wp_register_style( 'noticons', plugins_url( 'assets/stylesheets/noticons.css', __FILE__ ), array(), '20150727' );
-			wp_register_style( 'dashicons', plugins_url( 'assets/stylesheets/dashicons.css', __FILE__ ), array(), '20150727' );
+			// Note: This will break outside of wp-admin, if/when we put user-facing JS/CSS we'll have to figure out another way to version them
```

Comment:
WooCommerce core uses a hard-coded `define( 'WC_VERSION', '2.7.0' )` constant for this, but as long as we only need to be inside wp-admin, there's no need for that.